source/Calamari/Deployment/Features/IisWebSiteBeforeDeployFeature.cs

```diff
@@ -57,7 +57,10 @@ static void EnsureCertificateInStore(IVariables variables, string certificateVar
                 storeName = AddCertificateToLocalMachineStore(variables, certificateVariable);
             }
 
-            Log.SetOutputVariable(SpecialVariables.Action.IisWebSite.Output.CertificateStoreName, storeName, variables);
+            var storeNamesVariable = variables.Get(SpecialVariables.Action.IisWebSite.Output.CertificateStoreName, "");
+            storeNamesVariable = string.Join(",", storeNamesVariable, storeName);
+
+            Log.SetOutputVariable(SpecialVariables.Action.IisWebSite.Output.CertificateStoreName, storeNamesVariable, variables);
```

Comment:
I haven't renamed this from CertificateStoreName -> CertificateStoreNames. This variable looks to be internal only  and changing from a single store name to a comma-delimited list is probably a larger change.  But I've left the variable name as is for consistency, happy to rename but figured it's a small thing either way